### DISCUSSION TITLE
How to configure more than 1 device

### DISCUSSION THREAD

**Discussion by JumpingJackFly** [Hyundai-Kia-Connect/kia_uvo#988] [Q&A] [answered]
Hi,

I monitor my HYUNDAI IONIQ5 since 2021 in Home Assistant and everything works like charm.

Today, in addition to this car, i'm about to buy a second one, a KIA NIRO EV and am wondering if there is a option/hack/tips/possibility to add it as well in my HA and be able to monitor the 2 cars in parallel.

Is there anyone who already did it ?

**Comment by cdnninja** [comment]
Yes.  Can either add to the existing account or a second account.

**Reply by JumpingJackFly** [reply]
All right, Thanks. Can you please guide on how to do it ? I didn't find any way when checking HA or Device or HACS settings sections. Should I uninstall and reinstall it from scratch?

**Reply by Esprit1st** [reply]
The second car should just show up once you have added it to your account (if you use one account for both cars). All HA instances of all switches, sensors etc. have the cars name in it, so you will just have the same sensor with the cars different name in it. It should be very straight forward.

**Reply by JumpingJackFly** [reply]
OK, got it. Thanks. I'm afraid this will not make the trick because my IONIQ5 is attached to a BlueLink account while my future NIRO EV will the attached to a KIA connect account. Although both brands are part of the same group, I doubt they share the same connectivity. Do they do it behind the scene?

**Reply by cdnninja** [reply, marked as answer]
That's fine.  Add the integration a second time and enter your other creds.

**Comment by JumpingJackFly** [comment]
Super! Will try for sure once the NIRO will be at home. Thanks for your prompt help and also for keep maintaining this amazing HA hack

**Reply by JumpingJackFly** [reply]
Hi,
Now this is in place, I confirm it works like a charm ! Thanks a lot for all the effort the team do to maintain this lib at the top :-)
